superset-frontend/src/cccs-viz/plugins/plugin-chart-at-a-glance-user-id/src/styles.js

```diff
@@ -70,7 +70,7 @@ const AtAGlance = {
 };
 
 const Table = {
-  width: '500px',
+  width: '600px',
```

Comment:
```suggestion
  width: '100%',
```
This will fix the weird scaling on larger screens